bin/rspecq

```diff
@@ -72,6 +72,11 @@ OptionParser.new do |o|
     opts[:max_requeues] = v
   end
 
+  o.on("--fail-fast N", Integer, "Stop running the suite after N failed tests " \
+       "(default: #{nil}).") do |v|
```

Comment:
Ditto regarding the message here.

Reply:
Is this still valid?

Reply:
Yes it is. This message should be updated as well.